control-plane/csi-driver/src/bin/controller/controller.rs

```diff
@@ -818,7 +818,7 @@ impl rpc::csi::controller_server::Controller for CsiControllerSvc {
 
 fn snapshot_to_csi(snapshot: models::VolumeSnapshot) -> Snapshot {
     Snapshot {
-        size_bytes: snapshot.state.size as i64,
+        size_bytes: 0,
```

Comment:
Please add a comment stating we should come back to this once we know what this size should be exactly.